libmscore/score.cpp

```diff
@@ -1761,13 +1761,17 @@ bool Score::appendScore(Score* score)
             return false;
       int tickOfAppend = lastMeasure->endTick();
 
-      if (!lastMeasure->lineBreak() && !lastMeasure->pageBreak()) {
-            lastMeasure->undoSetBreak(true, LayoutBreak::Type::LINE);
+      // apply Page/Section Breaks if desired
```

Comment:
@Jojo-Schmitz, I've changed around this logic regarding addPageBreak so is as follows.  If adding a pagebreak, will first remove a line break if one exists.  If not adding a page break, will simply add a line break (if a page break or line break doesn't already exist)...this is same behavior pre-PR.